routes/utils.js

```diff
@@ -65,10 +171,29 @@ function updateCurrentProjectFiles(config, token, session, project, callback) {
   });
 }
 
+function getRemixedProjectFiles(config, projectId, callback) {
+  var publishURL = config.publishURL + "/publishedProjects/" + projectId + "/publishedFiles";
+
+  request.get({ uri: publishURL }, function(err, response, body) {
+    if(err) {
+      console.error("Failed to send request to " + publishURL + " with: ", err);
+      callback(null, 500);
```

Comment:
Agreed. That said, I think I'd prefer fewer cases to more. Right now you have:

1. `err` exists
2. an error happened, but `err` is null
3. success.

Shouldn't 1 & 2 be combined somehow? If you passed a generic status it would solve the problem, and you would be able to get rid of the 

```js
if(status === 500)
```
case entirely.